models/_sector/nft/trades/old/platforms/stealcam_arbitrum_events.sql

```diff
@@ -1,7 +1,7 @@
 {{ config(
     schema = 'stealcam_arbitrum',
     alias = alias('events'),
-    partition_by = ['block_date'],
```

Comment:
also, how come we're removing partitions on the arbritrum spells?

Reply:
following this guideline on delta lake partitioning:
https://docs.databricks.com/tables/partitions.html#do-small-tables-need-to-be-partitioned
I'm going to remove partitioning from the smaller marketplace models. (this specific one is only 50k records)

Reply:
good point, we should eventually do this check universally. i'll add to list for post-migration.